internal/renderers/skia/lib.rs

```diff
@@ -339,7 +361,8 @@ impl i_slint_core::renderer::RendererSealed for SkiaRenderer {
         &self,
         callback: Box<dyn RenderingNotifier>,
     ) -> std::result::Result<(), SetRenderingNotifierError> {
-        if !self.surface.supports_graphics_api() {
+        // FIXME: true or false when the surface is not set?
+        if !self.surface.borrow().as_ref().map_or(true, |x| x.supports_graphics_api()) {
```

Comment:
This is disapointing that if the dev calls set_rendering_notifier before a Resume event, it will not be notified that the backend don't support it.
Perhaps it could be  `DefaultSurface::supports_graphics_api()`  (without self)  but then this would be wrong if the surface is not the default one

Reply:
As a best effort I think `DefaultSurface` is the way to go for now. In the future we might need to replace the `impl Surface` with something like a `impl SurfaceFactory`, and then the factory can report.